src/Klarna/Rest/InstantShopping/ButtonKeys.php

```diff
@@ -73,10 +74,12 @@ public function create(array $data)
     {
         $data = $this->post(self::$path, $data)
             ->status('201')
-            ->contentType('application/json')
-            ->getJson();
+            ->contentType('application/json');
```

Comment:
Please fix this variable ($data), I prefer the solution in line 99 where we don't overwrite it as it comes in as an argument.